inst/tests/tests.Rraw

```diff
@@ -10648,7 +10650,7 @@ test(1779.2, uniqueN(lt)==1L)
 # some random 1e6 timestamps old defaults vs new methods UTC
 intpx = function(x) as.integer(as.POSIXct(x, origin = "1970-01-01", tz = "UTC"))
 set.seed(1)
-i = sample(seq(intpx("2014-10-12"), intpx("2015-10-12")), 1e6, TRUE)
+i = sample(intpx("2015-10-12")-intpx("2014-10-12"), 1e5, TRUE) + intpx("2014-10-12")
```

Comment:
reduced data size and also removed generating 3M elements `seq` just to sample from it.

Reply:
Agreed.  Good.